apps/server/src/routes/projectRouter.ts

```diff
@@ -9,3 +16,12 @@ router.get('/', getProject);
 
 // create route between controller and 'POST /project' endpoint
 router.post('/', projectSanitiser, postProject);
+
+router.get('/custom', getCustomFields);
+
+//TODO: validate
+router.post('/custom', postCustomField);
```

Comment:
I understand the instinct to keep this brief, but `custom` is a very tricky keyword to understand. `field` would have not been munch better. could we call this endpoint `custom-field`?